prow/plugins/bugzilla/bugzilla.go

```diff
@@ -91,8 +90,9 @@ func helpProvider(config *plugins.Configuration, enabledRepos []string) (*plugin
 			if opts[branch].TargetRelease != nil {
 				conditions = append(conditions, fmt.Sprintf("target the %q release", *opts[branch].TargetRelease))
 			}
-			if opts[branch].Statuses != nil {
```

Comment:
So while we keep supporting the old fields we stop exposing them in plugin help? Probably fine if we make it obvious that someone needs to update their config

Reply:
Both `Statuses` and `States` are actually exposed here, as the former are converted to states and merged with `States` in the resolution done under `OptionsForRepo`.

Reply:
Both Statuses and States are actually exposed here, as the former are converted to states and merged with States in the resolution done under OptionsForRepo.